### DISCUSSION TITLE
【求助】问问各位神将怎么升级？

### DISCUSSION THREAD

**Discussion by 75020-maker** [share121/inter-knot#608] [其他]
第一：绳网信用在哪刷啊，新手不会弄啊QAQ
第二：没有代理人怎么办啊？狡兔屋联系不上啊，有人有途径吗？
第三：听其他绳匠说喝咖啡可以升级，是真的吗？
第四： Ciallo～(∠・ω< )⌒☆
![5f887a36139a4d6e9cec8d4741136a8](https://github.com/user-attachments/assets/7edf81d8-1d80-4e50-b417-f13296a837df)
第五......人格？启动！

**Comment by StarrySkyehk** [comment]
Ciallo～(∠・ω< )⌒☆

**Comment by ReALiv** [comment]
Ciallo～(∠・ω< )⌒☆
